src/contractWrappers/arbitrator/Arbitrator.js

```diff
@@ -0,0 +1,60 @@
+import ContractWrapper from '../ContractWrapper'
+import * as errorConstants from '../../constants/error'
+
+class ArbitratorContract extends ContractWrapper {
+  constructor(web3Wrapper, contractAddress, artifact) {
+    super(web3Wrapper)
+    this.contractAddress = contractAddress
+    this.artifact = artifact
+    this.contractInstance = null
+  }
+
+  /**
+   * Load an existing contract from the current arbitrator artifact and address
+   * @returns {object} - The contract instance.
+   */
+  _load = async () => {
+    try {
+      this.contractInstance = await this._instantiateContractIfExistsAsync(
+        this.artifact,
+        this.contractAddress
+      )
+
+      return this.contractInstance
+    } catch (err) {
+      console.error(err)
+      throw new Error(errorConstants.UNABLE_TO_LOAD_ARBITRATOR)
+    }
+  }
+
+  /**
+   * Checks if contractInstace is set. If it can it will load contract instance.
+   * If not throw an error.
+   * @returns {object} contractInstance object
+   */
+  _checkContractInstanceSet = async () => {
+    if (!this.contractInstance) {
+      if (this.contractAddress && this.artifact) return this._load()
+
+      throw new Error(errorConstants.ARBITRATOR_NOT_SET)
+    }
+  }
+
+  /**
+   * Set a new arbitrator
+   * @param {string} contractAddress - The address of the contract
+   * @param {object} artifact - Contract artifact to use to load contract
+   * @returns {object} contractInstance object
+   */
+  setArbitrator = async (contractAddress, artifact) => {
+    this.contractAddress = contractAddress || this.contractAddress
+    this.artifact = artifact || this.artifact
+    return this._load()
+  }
+
+  getContractInstance = () => this.contractInstance
```

Comment:
This is needed because at least at the moment abstractWrapper/arbitrator is only inheriting/delegating the functions from KlerosPOC. So there is no way to access it via the api without a getter

Reply:
You could keep a reference like before, `this.contractWrapper`.

Reply:
There is one but its private at the moment `this._contractWrapper`. It kind of defeats the purpose of the delegation if its public, as then all those methods can just be called via `arbitrator.contactWrapper....`, which maybe isn't a bad alternative to trying to be tricky with the delegation, but does require the consumer to have a deeper knowledge of where their calls live

Reply:
I am going to leave it as is for now